packages/grid/package.json

```diff
@@ -27,6 +27,9 @@
     "react": ">=16.8.0",
     "react-dom": ">=16.8.0"
   },
+  "devDependencies": {
+    "@zendeskgarden/container-utilities": "^0.5.6"
```

Comment:
This should be a direct dependency.

Reply:
I added this as a dev dependency because the exported utility (`convertToMatrix`) is only used in the Storybook demo and not required at runtime for the `grid` package.

Reply:
Your PR description sez

> It will also be used in react-components stories to demonstrate Garden's new color swatch component.

Reply:
Ah, nvmd my bad. Gotcha.